deploy/charts/rook-ceph/values.yaml

```diff
@@ -358,3 +358,18 @@ monitoring:
   # requires Prometheus to be pre-installed
   # enabling will also create RBAC rules to allow Operator to create ServiceMonitors
   enabled: false
+  # prometheus rule alerts values for overriding default prometheus rules values
+  # optional: specific alerts can be disabled by setting disabled field to true
+  #alertRuleOverrides:
+  #  CephNodeDown:
+  #    disabled: true
+  #  CephMgrIsAbsent:
+  #    for: 1m
+  #    severityLevel: warning # must be warning, critical, or error
+  #    severity: custom-severity
```

Comment:
Looks like there are a couple other example severities to update

Reply:
@travisn please see my [comment](https://github.com/rook/rook/pull/9503#discussion_r786311138) above.
I think we should keep the `severity` field customizable by user preference.
What do you think?